doc/rules/operator/operator_linebreak.rst

```diff
@@ -26,6 +26,15 @@ Allowed values: ``'beginning'``, ``'end'``
 
 Default value: ``'beginning'``
 
+``ignored``
```

Comment:
All the options are declaring what will be done (e.g. `allow_single_line_closure`), so let's have `ignore` here.